nemo/collections/common/tokenizers/sentencepiece_tokenizer.py

```diff
@@ -206,6 +260,11 @@ def create_spt_model(
             to build the tokenizer.
         max_sentencepiece_length: Limits the maximum length of the SentencePiece subword that can be constructed.
             By default, no limit is placed.
+        bos: when True, bos token "<s>" is added to the vocabulary.
+        eos: when True, eos token "</s>" is added to the vocabulary.
+        pad: when True, pad token "<pad>" is added to the vocabulary.
+        control_symbols: control symbols to add to tokenizer, as defined by sentencepiece.
```

Comment:
Document difference between `control_symbols` and `user_defined_symbols` - maybe with example.

Reply:
Added explanation for both in the docstring.